docs: Add deployed tags for local execution (#7082)
PrairieLearn/PrairieLearn: docs/installing.md

After this, run PrairieLearn using the same commands as above.

## Running a specific version of PrairieLearn

> The commands above will always run the very latest version of PrairieLearn, which might be an unreleased development version. If you would like to run the version that is currently deployed (or a specific older version), use a tag associated to the environment your course is running with. For PrairieLearn's main servers, the tags `us-prod-live` and `ca-live` match the deployed versions in the US and Canadian servers, respectively.

Reviewer: ```suggestion
The commands above will always run the very latest version of PrairieLearn, which might be an unreleased development version. If you would like to run the version that is currently deployed (or a specific older version), use the appropriate tag for the server you're using:

- https://us.prairielearn.com: `us-prod-live`
- https://ca.prairielearn.com: `ca-live`
```

Author: I massaged the content above and added a note about other institutions in e172060b7.

```sh
docker run -it --rm -p 3000:3000 [other args] prairielearn/prairielearn:us-prod-live
```

> Make sure to also upgrade the Docker version periodically. Updates in the production servers are announced in the [PrairieLearn GitHub Discussions page](https://github.com/PrairieLearn/PrairieLearn/discussions/categories/announcements). When an announcement is made, you are encouraged to run:

Reviewer: ```suggestion
Make sure to periodically pull new versions of the image. Updates in the production servers are announced in the [PrairieLearn GitHub Discussions page](https://github.com/PrairieLearn/PrairieLearn/discussions/categories/announcements). When an announcement is made, you are encouraged to run:
```

Alternatively, we could update the above command to use [`--pull=always`](https://docs.docker.com/engine/reference/commandline/run/#-set-the-pull-policy---pull)?

Author: I changed to `--pull=always` and replaced this statement with a request to restart instead of pull in 9bc2e03.

```sh
docker pull prairielearn/prairielearn:us-prod-live
```

> Additional tags are available for older versions. The list of available versions is viewable on the [hub.docker build page](https://hub.docker.com/r/prairielearn/prairielearn/builds/).

Reviewer: ```suggestion
Additional tags are available for older versions. The list of available versions is viewable on the [Docker Hub build page](https://hub.docker.com/r/prairielearn/prairielearn/builds/).
```
